src/bin/rawst.rs

```diff
@@ -1,9 +1,45 @@
-use rawst::cli::parser::init;
+use rawst::cli::args;
+use rawst::cli::args::Command;
+use rawst::cli::parser as not_the_parser;
```

Comment:
I didn't want to move the file on the same commit to keep things simple.
However, I felt I had to alias this module as it really isn't the parser and instead has a lot of application logic and implements download, download list, resume, and history.

Reply:
how about "service" instead of "parser"? or do you have any other suggestions?